### ISSUE TITLE
Builds getting catapulted out of sight while building

### ISSUE THREAD

**Issue by Arne2512** [tsunamayo/Starship-EVO#1976]
### Short Description of the issue:

While building a ship and taking away the startblock, the unfinished ship dropes to the floor. If you ad tiles then, the whole build gets catapulted out of sight.

### Steps to reproduce the issue:

Build a Starter Block. Add a wall of Hull Basic Blocks (1)
![1](https://user-images.githubusercontent.com/7555381/81110382-c4fc9d00-8f1b-11ea-9176-a2c71e9c1ca1.png)
Remove the Starter Block - Hull Basic block Wall falls to ground (2)
![2](https://user-images.githubusercontent.com/7555381/81110395-c928ba80-8f1b-11ea-9b9c-52406c6fe6b3.png)
Add some more Hull Basic Blocks and watch it get catapulted away (3)
![3](https://user-images.githubusercontent.com/7555381/81110404-cc23ab00-8f1b-11ea-878b-88e3c9677d0a.png)

### Environment
- Starship EVO version: pre-alpha 20w19b
- Graphics Card: NVIDIA GeForce GTX 1070

### Additional information:

[output_log.txt](https://github.com/tsunamayo/Starship-EVO/files/4582872/output_log.txt)

**Comment by Aiwii** [comment]
The build experiencing gravity when the starter block is removed is intended. Not sure about the flinging...

**Comment by tsunamayo** [comment]
Yes the gravity fall is intended, but not the catapult of course. I never saw that, @Arne2512 is this something you can reproduce often? Thanks

**Comment by Arne2512** [comment]
The Gravity fall is absolutely fine, but the catapult seems not to be wanted.
Yes, I can reproduce this again and again. And I think I found the Problem.
If the Build lies flat on the Surface, and you add tiles, the tiles reach beyond the surface (can't see it at that moment) and that starts the "shot"
I looked at the catapulted pieces an found that every new added tiles are bigger (not 1.00 1.00 1.00) blocks. So this is a collision problem with the surface that causes the Catapult.

**Comment by YnotnA** [comment]
I reproduce the bug systematically. It happens when you place blocks that come to touch the ground on an existing structure. https://youtu.be/pCBfut8RUnY
